.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: ''
+    hooks:
+    -   id: end-of-file-fixer
+        exclude: '(?:.*\.(ai|txt)|extern/*)'
+    -   id: trailing-whitespace
+        exclude: '(?:.*\.(ai|txt)|extern/*)'
+    -   id: debug-statements
+-   repo: local
+    hooks:
+    -   id: flake8
+        name: flake8
+        entry: flake8
+        language: python
+        types: [python]
+        args:
+        - --show-source
```

Comment:
Can this option be configured with `setup.cfg` or `pyproject.toml` instead?

Reply:
It can, but that would affect all runs of `flake8` in the repo, including manual user runs. Is that really what we want? We don't do that her or for any other repo because it gets pretty verbose and makes it harder to work with when you need to manually go through and fix many things.

Reply:
That's fine, good point -- marking as resolved.